doc/userguide/rules/dns-keywords.rst

```diff
@@ -176,3 +176,31 @@ resource name, for example "www.suricata.io".
 ``dns.query.name`` supports :doc:`multi-buffer-matching`.
 
 ``dns.query.name`` was introduced in Suricata 8.0.0.
+
+dns.response.rrname
+-------------------
+
+``dns.response.rrname`` is a sticky buffer that is used to look at all name
+and rdata fields of DNS response (answer) resource records that are
+represented as a resource name (hostname). It supports inspecting all
+DNS response sections. Example::
+
+  alert dns any any -> any any (msg:"Test dns.response.rrname option"; \
+      dns.response.rrname; content:"google"; nocase; sid:1;)
+
+``rdata`` field matching supports a subset of types that contain
+domain name structured data, for example: "www.suricata.io".  The list
+of types inspected is:
```

Comment:
Could we have an example with ``rdata``, too?

Other than that, doc looks good to me :)

Reply:
`dns.reponse.rrname` matches on rdata that is formatted a a resource name record. There is no additional keyword.